pyproject.toml

```diff
@@ -76,6 +77,7 @@ tests = [
     "pytest-watch",
     "tox",
 ]
+asyncio = ["asyncpg"]
```

Comment:
are we assuming that sqla is a peer dependency (i.e. must be installed by consumer), or is it a true dependency? If the former, then is it fastapi-sqla's job to include this?

Reply:
I honestly do not know. Same concern with sqlalchemy IIUC. What is your opinon?

Reply:
Let's keep it as an extra for now :) poetry / setuptools doesn't have a concept of peer dependencies AFAIK, so this is fine as long as documented